G1ANT.Addon.IExplorer/G1ANT.Addon.IE.Tests/Commands/ie.attach/ShouldNotChangeTitle_WhenReattachedByPhrase.G1ANT

```diff
@@ -0,0 +1,11 @@
+﻿addon iexplorer version 2.205.17304.1000
+addon core version 4.99.18340.1354
+addon language version 4.99.18340.1354
```

Comment:
I think you don't have to add core & language anywhere
![](https://media.giphy.com/media/ANbD1CCdA3iI8/giphy.gif)